src/components/ValueSet/ValueSetEditor.tsx

```diff
@@ -0,0 +1,112 @@
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { useNavigate } from "raviger";
+import { toast } from "sonner";
+
+import { Skeleton } from "@/components/ui/skeleton";
+
+import mutate from "@/Utils/request/mutate";
+import query from "@/Utils/request/query";
+import {
+  CreateValuesetModel,
+  UpdateValuesetModel,
+  ValuesetBase,
+  ValuesetFormType,
+} from "@/types/valueset/valueset";
+import valuesetApi from "@/types/valueset/valuesetApi";
+
+import { ValueSetForm } from "./ValueSetForm";
+
+interface ValueSetEditorProps {
+  slug?: string; // If provided, we're editing an existing valueset
+}
+
+function FormSkeleton() {
```

Comment:
Skelton should be part of the skeltonloder

Reply:
Moved to SkeletonLoading